apps/voting/contracts/Voting.sol

```diff
@@ -117,6 +128,19 @@ contract Voting is IForwarder, AragonApp {
         emit ChangeMinQuorum(_minAcceptQuorumPct);
     }
 
+    /**
+    * @notice Change overrule window to a duration of `@transformTime(_newOverruleWindow)`
+    * @param _newOverruleWindow New overrule window in seconds
+    */
+    function changeOverruleWindow(uint64 _newOverruleWindow)
```

Comment:
Oh sorry, I missed that comment. Actually we didn't discussed that, I actually see it as feature, you can "disable" proxy voting by setting a large overrule window for example

Reply:
Yes, sure, but I would see at "init" time. The problem would be to change it just before creating a vote or with a vote ongoing.

Reply:
I don't see why not allowing that behavior, I don't see this as a core setting that should be one-shot as the voting time for example. Do you see any inconsistencies in case we allow that?